lua/telescope/builtin/internal.lua

```diff
@@ -560,15 +560,36 @@ internal.buffers = function(opts)
   }):find()
 end
 
+local colorscheme_previewer = previewers.new({
+	setup = function (self)
+		-- save original colorscheme
+		vim.api.nvim_err_writeln("Ln")
+	end,
+	preview_fn  = function(self, entry, status)
```

Comment:
`preview_fn` will run on selecting a entry. That is correct. You have `colors` commented out which means a entry will never be selected